Content.Client/Voting/VoteManager.cs

```diff
@@ -115,6 +117,7 @@ private void ReceiveVoteData(MsgVoteData message)
                 }
 
                 @new = true;
+                SoundSystem.Play(Filter.Local(), "/Audio/Effects/voteding.ogg");
```

Comment:
pretty sure this only plays the sound to the player that calls the vote, since you're using Filter.Local()

Reply:
But it's in `ReceiveVoteData()`